### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by zobe123**

### CODE REVIEW

File: data/interfaces/default/settings.html [hunk 1]
```diff
@@ -1359,7 +1359,7 @@ <h4 class="modal-title">Notification String Substitutions</h4>
                                 </tr>
                                 <tr>
                                     <td><strong>{rating_key}</strong></td>
-                                    <td>The unique identifier for the item.</td>
+                                    <td>The unique identifier for the item (Episode).</td>
```
Comment (JonnyWong16) [inline data/interfaces/default/settings.html:1362]: It's actually Movie, Episode, or Track.

File: data/interfaces/default/settings.html [hunk 2]
```diff
@@ -61,7 +61,7 @@
                             <p class="help-block">If you have Git installed, allow periodic checks for updates.</p>
                         </div>
                         % if plexpy.CURRENT_VERSION:
-                        <p class="help-block">Git hash: ${plexpy.CURRENT_VERSION}</p>
+                        <p class="help-block">Git hash: ${plexpy.CURRENT_VERSION}</br>Git branch: ${plexpy.CONFIG.GIT_BRANCH}</p>
```
Comment (JonnyWong16) [inline data/interfaces/default/settings.html:64]: Let's put branch before hash.

**Comment by zobe123** [comment]
Should i also put repository link there (#490) ?

**Comment by JonnyWong16** [comment]
Added in v1.3.7.

**Pull request closed without merging**